roles/micado_master/build/tasks/core-install.yml

```diff
@@ -42,6 +42,7 @@
   - { src: 'iptables/ip6tables.service', dest: '/etc/systemd/system/ip6tables.service', mode: '0644'}
   - { src: 'micado/micado.service', dest: '/etc/systemd/system/micado.service', mode: '0644'}
   - { src: 'micado/micadoctl', dest: '/usr/local/bin/micadoctl', mode: '0755'}
+  - { src: 'terraform/terraform', dest: '/usr/local/sbin/terraform', mode: '0755'}
```

Comment:
We can do that. But I though it was better to keep it in a binary folder itself. But we can actually mount the file alone rather than the whole directory.

Reply:
I agree with Jay, the path should contain information that it is a wrapper...

Reply:
I will change it to the one Jay suggested.